### PR TITLE
Add mobile version

### PR DESCRIPTION
**Pull request by rotimiazeez**
# Portfolio Page: Mobile Version 🎉 

> This PR describes the second milestone of the portfolio project. This project is built with a mobile-first approach with the page mobile section built first. 💪🏽 

Additional description of the project and its features. ➕ 

- Use Flexbox and Grid to layout elements in the "Work" section.
- Use `ul `and `li `for categories in the "About myself" section. Languages, Frameworks, and Skills are list items.
- Build the"Work" section
- Build the "About myself" section.

## Built With ❤️ 

- HTML (HTML5 semantic tags, `ul`, `li`,  `button`)
- CSS ( `Flexbox`, `Grid`, `background-images` etc. )

## Deployed Preview 🚀 
[Live Preview](https://deploy-preview-2--rotimiazeez-portfolio.netlify.app/)

### NOTE TO REVIEWER 😉 ❤️  

- For best result and viewing experience, kindly use the developer tool mobile simulators 😃 👍🏽

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -49,6 +49,212 @@ <h1 class="pad-top-158 font-size-40 font-weight-800 line-height-52 slategray mar
             <li><a href="#" class="margin-left-16"><img src="/assets/angelist.svg" alt="angellist icon"></a></li>
         </ul>
     </section>
+
+    <div class="grid d-grid">
```
Comment (Lameck1) [inline index.html:53]: - [ ] The container of the works section should be a `<section>`  tag not a `<div>` tag. Please correct this.

File: index.html [hunk 2]
```diff
@@ -49,6 +49,212 @@ <h1 class="pad-top-158 font-size-40 font-weight-800 line-height-52 slategray mar
...
+                            html</li>
+                        <li
+                            class="font-size-12 font-weight-500 background-grey line-height-20 dodgerblue margin-left-8 height-20 border-radius-7 width-45 pad-left-12">
+                            css
+                        </li>
+                        <li
+                            class="font-size-12 font-weight-500 background-grey line-height-20 dodgerblue margin-left-8 height-20 border-radius-7 width-65 pad-left-7">
+                            javascript</li>
+                    </ul>
+                    <button type="button"
```
Comment (Lameck1) [inline index.html:82]: - There is no button interaction implemented on any of the buttons on the page. Please work on this. It is suggested by the requirement screenshot below:

    requirement

**Review by Lameck1 — CHANGES_REQUESTED** [review]
Hello @rotimiazeez, 

Splendid job ⭐. It's evident you've put a lot of effort into this second milestone.

Let us revisit some areas that still need our attention so that we can move on to the next task.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (rotimiazeez) [reply index.html:53]: Thanks for pointing that out it has been changed.

File: index.html (hunk 2 above)
Comment (rotimiazeez) [reply index.html:82]: 👍🏽

**Comment by rotimiazeez** [comment]
Hello @Lameck1 😄, nice to have you on my PR again 👍🏽. 
I have made the suggested changes. I was carried away that why I missed them. 🚀 
I hope this gets approved now. 💪🏽

**Review by mikenath223 — APPROVED** [review]
Hello there  

Your project is complete!  There is nothing else to say other than... it's time to merge it 🛳 
Congratulations! 🎉 

Happy coding. Cheers! 🚀🚀

If you feel something isn't clear enough you can leave a question or comment in the PR thread.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by rotimiazeez** [comment]
Hi @mikenath223 👋🏽,
Thank you for the approval. 👍🏽 That will be all for now. Happy coding. Cheers! 🚀🚀

**Pull request merged**